cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -393,7 +403,16 @@ func main() {
 
 	klog.V(1).Infof("GPU device plugin started with %s preferred allocation policy", opts.preferredAllocationPolicy)
 
-	plugin := newDevicePlugin(sysfsDrmDirectory, devfsDriDirectory, opts)
+	var sysfs, devfs string
```

Comment:
Too much code here. If prefix equals "" then adding it in front doesn't harm anything. Just do:
sysfs := prefix + sysfsDrmDirectory
devfs := prefix + devfsDriDirectory

Reply:
or just dump it in the function call args

Reply:
Done. Now with all remains of -fake-mode stuff cleaned, option addition is really trivial.